# 036 인덱싱 에러
#
# 다음 코드의 결과를 예상해보고 에러가 발생한 이유를 설명하라.
#
# >>> t = 'python'
# >>> t[100]
# Traceback (most recent call last):
#   File "<pyshell#339>", line 1, in <module>
#     t[100]
# IndexError: string index out of range
# >>>
##인덱스 범위 초과